lnwire/open_channel2.go

```diff
@@ -79,11 +80,9 @@ type OpenChannel2 struct {
 	// revocation key for the commitment transaction.
 	RevocationPoint *btcec.PublicKey
 
-	// PaymentPoint is the base payment point for the sending party. This
-	// key should be combined with the per commitment point for a
-	// particular commitment state in order to create the key that should
-	// be used in any output that pays directly to the sending party, and
-	// also within the HTLC covenant transactions.
+	// PaymentPoint is the base payment point for the initiating party, used
```

Comment:
I think this comment is not 100% correct, I think the PaymentPoint is currently only used for the to_remote output of the receiving node commitment transaction, or in other words in the to_remote for `their` commitment transaction. Maybe something like this:
`The PaymentPoint is used for outputs in the commitment transaction paying directly (no time-delay) to the sending party. Depending on the option `static_remote_key` this will be tweaked by the per commitment point. An example for this is the `to_remote` output of the receiving parties commitment transaction`

Reply:
Thanks, I like your wording.  Done.